gtk/src/default/gtk-3.20/_headerbar.scss

```diff
@@ -1,7 +1,7 @@
 
-$_headerbar_button_bg: lighten($headerbar_bg_color, 6%);
-$_dark_border_color: rgb(20, 19, 19);
-$_btn_backdrop_border: $headerbar_bg_color;
+$_headerbar_button_bg_color: lighten($headerbar_bg_color, 6%);
```

Comment:
I know this has started by me, however, since we are inside the `_headerbar.scss` file, I would drop the *_headerbar* prefix everywhere, it is just superfluous. Eventually, same story for the *_color* postfix for background variables, since it can only be a color.

Reply:
I agree on the headerbar suffix
But I would keep
Border
bg
or
fg

Reply:
It could also be a whole background just like in drawing

Reply:
border, bg and fg are correct, keep them, while for background it's not used anywhere as whole background here, so I don't expect misunderstandings